.gitignore

```diff
@@ -19,4 +19,6 @@ msbuild.wrn
 # Project specific
 F1WM/version.json
 F1WM/logs/*
-docker.env
\ No newline at end of file
+docker.env
+.DS_Store
```

Comment:
It's not project specific, it may be misleading. Please put these lines somewhere else